using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -6,7 +6,7 @@
 {% tab title="Android" %}
 This API appends Adobe visitor information to the query component of the specified URL.
 
-If the provided URL is null or empty, it is returned as is. Otherwise, the following information is added to the query component of the specified URL and is returned in the `AdobeCallback` instance:
+If the provided URL is null or empty, it is returned as is. Otherwise, the following information is added to the query component of the specified URL and is returned in the [AdobeCallback](../../mobile-core-api-reference#adobecallback) instance:
```

Comment:
I'm using relative links here are that makes sense for portability, but I vaguely remember using absolute links is preferred. Let me know if I should change these links.

Reply:
Sometimes the relative links don't work, let's use the absolute ones everywhere

Reply:
👍 I fixed some of the links to Analytics as well.